tests/test_notebooks.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Están curiosos este tipo de archivos o código similar. Aquellos en los que se ve que la manera en el que lo estamos haciendo no es la correcta, pero el resultado es el esperado. Es decir, seguro debería de haber una manera mejor de saber si corre o un _notebook_ pero no la sabemos. Pero sí sabemos programar algo como para averiguarlo. 

Está curioso, porque me recuerda a la vida antes de los _dataframe_. Todo lo hacemos con las herramientas básicas `for` e `if` sin utilizar las herramientas que ya están hechas para eso. 

Vientos @devarops que bien que ya tengamos una opción de cómo resolver ese detalle.

Reply:
Estoy de acuerdo. Yo también siendo que estamos reinventando la rueda. Seguramente hay una mejor manera de hacer esto. Pero actualmente no la he encontrado.

Este PR no está listo para su revisón; aún estoy trabajando en él.